### PR TITLE
Update bucket

### PR DESCRIPTION
**Pull request by jarqvi**

### CODE REVIEW

File: src/commands/bucket/create.ts [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+  async setGotConfig(
+    config: IConfig,
+    isObjMode: boolean = true
+  ): Promise<void> {
+    if (isObjMode) {
+      await super.setGotConfig(config);
+      this.got = this.got.extend({
+        prefixUrl: DEV_MODE
+          ? 'http://localhost:3000'
```
Comment (mhemrg) [inline src/commands/bucket/create.ts:37]:
```suggestion
          ? 'http://localhost:3001'
```

File: src/constants.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export const GLOBAL_CONF_VERSION = '1';
 export const REGIONS_API_URL: { [key: string]: string } = {
   iran: 'https://api.iran.liara.ir',
   germany: 'https://api.liara.ir',
+  objStorage: 'https://storage-service.iran.liara.ir',
```
Comment (mhemrg) [inline src/constants.ts:13]: Please move this to a separate constant called `OBJECT_STORAGE_API_URL`. You can also add another constant for `OBJECT_STORAGE_API_URL_DEV`.

File: src/commands/bucket/delete.ts [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  async run() {
+    const { flags } = await this.parse(BucketDelete);
+    const debug = createDebugLogger(flags.debug);
+
+    await this.setGotConfig(flags);
+
+    const bucket = flags.bucket || (await this.promptBuckets());
+
+    try {
+      // TODO: Add --force or -f flag to force the deletion
```
Comment (mhemrg) [inline src/commands/bucket/delete.ts:39]: Please add this flag.

Comment (jarqvi) [reply src/commands/bucket/delete.ts]: Done.

File: src/constants.ts [hunk 4]
```diff
@@ -12,6 +12,14 @@ export const REGIONS_API_URL: { [key: string]: string } = {
...
 };
 
+export const OBJECT_STORAGE_API_URL: { [key: string]: string } = {
+  prodUri: 'https://storage-service.iran.liara.ir',
+};
+
+export const OBJECT_STORAGE_API_URL_DEV: { [key: string]: string } = {
+  devUri: 'http://localhost:3001',
+};
+
```
Comment (mhemrg) [inline src/constants.ts:22]:
```suggestion
export const OBJECT_STORAGE_API_URL = 'https://storage-service.iran.liara.ir';

export const OBJECT_STORAGE_API_URL_DEV=  'http://localhost:3001';

```

**Comment by jarqvi** [comment]
Done.

**Comment by mhemrg** [comment]
Awesome man!

**Pull request merged**